contents/customers/rayfit.md

```diff
@@ -0,0 +1,60 @@
+---
+title: How Rayfit uses PostHog AI to make analytics self-serve
```

Comment:
See my comments further into the piece itself before judging this 😄  (and feel free to disagree!)

```suggestion
title: How Rayfit uses PostHog AI to make all their data accessible
```